changelog/1.36.0.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "1.36.0"
+description: "Released on 10/26/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Added sessionAffinity to coderd service.
+- infra: Allowed older verions of curl for self-contained builds.
+- infra: Allowed workspace template builds even if user does not have permission
+  to add webhooks on their git repo.
+- infra: Broke out API request latency Prometheus metrics by route.
+- infra: Added Prometheus metrics around background job execution.
+- infra: Added support for cached CVMs with 5.15 and 5.16 kernel versions.
+
+### Bug fixes 🐛
+
+- web: Fixed the "back to site" button that appears when trying to access an IDE
+  on a workspace that has shut off.
+- infra: Fixed a goroutine leak.
+- web: Fixed relative links on embedded docs.
+- infra: Large numbers of concurrent workspace builds are distributed more
+  evenly amongst multiple coderd replicas.
+- web: Allow org members to view activity metrics.
+- infra: Prevent a panic when updating a template.
+- infra: Return the updated user after an update instead of the cached user.
```

Comment:
Idk if anyone relied on this? We can probably not include this. Might look better to have less "bugs". It was causing a unit test failure.
```suggestion
```